<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleTable</class>
 <widget class="QWidget" name="RoleTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="roleTableBg" native="true">
     <property name="styleSheet">
      <string notr="true">#roleTableBg{ border-radius : 10px; background-color : #FFFFFF; }
*{ font-size : 14px; font-family : 微软雅⿊; }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="inputArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#inputArea{ border-radius : 10px; background-color : #F7F8FA; }</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>41</number>
         </property>
         <property name="leftMargin">
          <number>23</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>27</number>
         </property>
         <item>
          <widget class="QWidget" name="adminPhoneBox" native="true">
           <property name="minimumSize">
            <size>
             <width>345</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>345</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="phoneLabel">
              <property name="text">
               <string>用户手机号：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="phone">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#phone{ 
	background : #FFFFFF; 
	border-radius : 4px; 
	border : 1px solid #DCDEE0; 
	padding-left : 18px; 
}</string>
              </property>
              <property name="placeholderText">
               <string>请输入手机号：</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="adminStatusBox" native="true">
           <property name="minimumSize">
            <size>
             <width>297</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>297</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="status">
              <property name="minimumSize">
               <size>
                <width>74</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>74</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>用户状态：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="userStatus">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#userStatus { 
	background-color: #FFFFFF; 
	border: 1px solid #DCDEE0; 
	border-radius: 4px; 
	color: #222222; 
	padding-left: 16px; 
	line-height: 40px; 
}
#userStatus::drop-down { 
	border: none; width: 33px; 
} 
#userStatus::down-arrow { 
	width: 15px; 
	background-color: transparent; 
	image: url(:/images/admin/triangle.png); 
} 
#userStatus QAbstractItemView { outline: 0px; } 
#userStatus QAbstractItemView::item { 
	color: #222222; 
	padding-left:16px; 
	background-color: #FFFFFF; 
	height: 30px; 
}
#userStatus QAbstractItemView::item:selected { 
	background-color: #409CE1; 
	color: #FFFFFF; 
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>258</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="btnOperatorBox" native="true">
           <property name="minimumSize">
            <size>
             <width>194</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>194</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="resetBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#resetBtn{ 
	color : #222222; 
	background-color: #FFFFFF; 
	border-radius: 4px; 
	border: 1px solid #DCDEE0; 
}</string>
              </property>
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="queryBtn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#queryBtn{ 
	color : #FFFFFF; 
	background-color: #FF6699; 
	border-radius: 4px; 
}</string>
              </property>
              <property name="text">
               <string>查询</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="insertBtnArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>74</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>74</height>
         </size>
        </property>
        <widget class="QLabel" name="addLabel">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>33</y>
           <width>12</width>
           <height>12</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#addLabel{ border-image : url(:/images/admin/add.png); }</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="insertBtn">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>19</y>
           <width>100</width>
           <height>40</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#insertBtn{ 
	color : #FF6699; 
	background-color : #FFFFFF; 
	border-radius: 8px; 
	border: 1px solid #FF6699; 
	padding :10px 25px 10px 47px; 
}</string>
         </property>
         <property name="text">
          <string>新增</string>
         </property>
        </widget>
        <zorder>insertBtn</zorder>
        <zorder>addLabel</zorder>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tableTitleArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>52</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#tableTitleArea{ 
	background-color: #FFECF1; 
	border-top-left-radius: 10px; 
	border-top-right-radius: 10px; 
	border-bottom-left-radius: 0; 
	border-bottom-right-radius: 0; 
} 
#tableTitleArea QLabel { 
	font-weight: bold; 
	color: #222222; 
	line-height: 20px; 
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="idLabel">
           <property name="minimumSize">
            <size>
             <width>174</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>174</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>序号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="roleLabel">
           <property name="minimumSize">
            <size>
             <width>174</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>174</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>用户角色</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="telphoneLabel">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>手机号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>用户昵称</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="minimumSize">
            <size>
             <width>188</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>188</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>状态</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="commentLabel">
           <property name="minimumSize">
            <size>
             <width>188</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>188</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>备注</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="operationLabel">
           <property name="text">
            <string>操作</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="adminInfoArea" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>408</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>408</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">#scrollArea { border : none; background-color: #FFFFFF; }
QScrollBar:vertical{ /*垂直滚动条*/ 
	border : none; 
	background-color : #FFFFFF; 
	width : 10px; 
} 
QScrollBar::handle:vertical{ /*设置垂直滚动条⼿柄*/ 
	border : none; 
	border-radius : 5px; 
	background-color : #E8E8E8; 
	min-height: 134px; 
} 
QScrollBar::up-arrow:vertical, QScrollBar::down- arrow:vertical { 
	border:none; 
	width: 0px; 
	height: 0px; 
} 
QScrollBar:horizontal { height: 0px; }</string>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1270</width>
              <height>393</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="layout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>11</number>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="PaginatorArea" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
